pyproject.toml

```diff
@@ -78,7 +78,7 @@ drf-nested-routers = "^0.93.5"
 django-scim2 = "^0.19.1"
 django-oauth-toolkit = "^2.3.0"
 youtube-transcript-api = "^0.6.2"
-ruff = "0.3.4"
```

Comment:
was removing ruff 0.3.4 intentional?

Reply:
think this was a rebase error... I put it back but looking at it I'm not sure why it's here and in dev dependencies (with a different version?) but that's sort of out of scope for this PR.

Reply:
I believe it's in there for IDE support. pre-commit is the source of truth for linting/formatting, but IDEs (to my knowledge) can't run pre-commit's ruff.

It's sometimes a bit out of date with pre-commit's, but they both get updated (pre-commit by its autoupdate, dev deps by renovate) and if they're slightly out of sync, it's not a big deal.